testing/integration/utxo_index_test.go

```diff
@@ -70,21 +70,11 @@ func TestUTXOIndex(t *testing.T) {
 		}
 	}
 
-	// Mine a block to include the above transaction
-	mineNextBlock(t, kaspad)
-	notification := <-onUTXOsChangedChan
-	if len(notification.Removed) > 0 {
-		t.Fatalf("Unexpectedly received that a UTXO has been removed")
-	}
-	for _, added := range notification.Added {
-		notificationEntries = append(notificationEntries, added)
-	}
-
-	// Mine another block to accept the above block
```

Comment:
Why did you remove this?

Reply:
Because the notification are now based on the virtual and not on the selected tip, so you don't need to mine extra block to get the expected notification